server/zanata-war/src/test/java/org/zanata/transaction/TransactionUtilForUnitTest.java

```diff
@@ -6,21 +6,25 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.zanata.arquillian.Deployments;
 import org.zanata.util.RunnableEx;
 
 /**
- * A TransactionUtil implementation used in tests.
+ * A TransactionUtil implementation used in unit tests. Arquillian deployment
+ * will exclude it from the deployment as the class name ends with Test.
+ *
+ * @see Deployments#notUnitTest(org.jboss.shrinkwrap.api.ArchivePath)
  */
-public class TestTransactionUtil implements TransactionUtil {
+public class TransactionUtilForUnitTest implements TransactionUtil {
```

Comment:
This whole naming convention we have for unit vs integration tests is a bit of a house of cards, but so far so good...